bgc_data/combined_bgc_map.sql

```diff
@@ -5,8 +5,8 @@ CREATE MATERIALIZED VIEW combined_bgc_map AS
     "Project",
     "StationName",
     "TripCode",
-    "SampleTime_local"::date AS "SampleDate_Local",
---  TODO: "SampleTime_UTC",
+    "SampleTime_local"::date::text AS "SampleDate_Local",
```

Comment:
For date I don't think we need to cast to `text` as Geoserver shouldn't add a `Z` to make it look like it's UTC (but we should check this in the stack).

Reply:
Correct, fixed in last commit - I double checked this in my stack and there's no added `Z` to either `SampleTime_local` or `SampleTime_UTC`